README.adoc

```diff
@@ -1 +1,20 @@
-= private-cloud
\ No newline at end of file
+= MuleSoft Contributor’s Agreement
+
+In consideration for the potential acceptance of Your Contributions in the MuleSoft Product you agree to the following terms and conditions (the “Agreement”):
+
+1. "Your Contributions" means all of your past, present, and future contributions of object code, source code and documentation to MuleSoft, however submitted to MuleSoft, excluding any submissions that are conspicuously marked or otherwise designated in writing by you as "Not a Contribution." “MuleSoft Products” means the proprietary MuleSoft web-based or software products including documentation. 
+
+1. You assign all worldwide right, title, and interest to Your Contributions in copyrights and related moral rights to MuleSoft. MuleSoft shall be able to register this assignment. However, at the same time, MuleSoft grants to you a non-exclusive, irrevocable, worldwide, royalty free, transferable copyright license to reproduce, prepare derivative works of, distribute (internally and externally, in object code and, if included in Your Contributions, source code form), use, publicly perform and publicly display Your Contributions, with the right to sublicense all of these rights through multiple tiers of sublicensees. The intention of the parties is that this license will be as broad as possible and to provide you with rights as similar as possible to the rights of the owner of the copyright. This license is limited to Your Contributions and does not provide any rights to the MuleSoft Products. You also grant to MuleSoft a non-exclusive, irrevocable, worldwide, royalty free, transferable license under any patents owned by You now or in the future to make, use, sell, offer for sale and import in Your Contributions, with the right to sublicense all of these rights through multiple tiers of sublicensees.
+
+1. You represent that you are legally entitled to assign the rights and grant the license set forth in paragraph 2. If your employer(s) have rights to intellectual property that you create, you represent that you have received permission to make Your Contributions on behalf of that employer, or that your employer has waived such rights for Your Contributions to MuleSoft. You further represent that you understand that your participation in this project shall be disclosed to other participants in the project.
+You represent that, except as disclosed in Your Contributions submission(s), each of Your Contributions is your original creation. You represent that Your Contributions submission(s) include complete details of any license or other restriction (including, but not limited to, related patents and trademarks) associated with any part of Your Contribution(s) (including a copy of any applicable license agreement). You agree to notify MuleSoft of any facts or circumstances of which you become aware that would make your representations in this Agreement inaccurate in any respect.
```

Comment:
I see that line 9 and line 10 are the only two parts on separate lines without a blank line between them. As long as that's correct, everything here looks great.